pom.xml

```diff
@@ -99,11 +99,6 @@ THE SOFTWARE.
     <!-- Make sure to keep the jetty-ee9-maven-plugin version in war/pom.xml in sync with the Jetty release in Winstone: -->
     <winstone.version>8.1</winstone.version>
     <node.version>20.17.0</node.version>
-    <!-- frontend-maven-plugin will install this Yarn version as bootstrap, then hand over control to Yarn Berry. -->
-    <yarn.version>1.22.19</yarn.version>
-    <!-- maven-antrun-plugin will download this Yarn version. -->
-    <yarn-berry.version>4.5.0</yarn-berry.version>
-    <yarn-berry.sha256sum>cc00dce5de4f68d11450519a0f69eadf2a1cbe5cc0d8e740bfac817a31d76874</yarn-berry.sha256sum>
```

Comment:
This duplicate information is no longer needed, as this information is consumed from `package.json` via Corepack.